app/assets/javascripts/selectors/index.js

```diff
@@ -1,11 +1,28 @@
 import { createSelector } from 'reselect';
-import { sortBy, difference, uniq, filter, includes, pickBy, find } from 'lodash-es';
+import {
```

Comment:
I'd prefer not to include all these formatting changes alongside the new feature.